packages/lib/src/views/RolesView.tsx

```diff
@@ -110,9 +110,9 @@ export function RolesView(): JSX.Element {
 
   const { data: rights } = useGetRights({ page: 0, size: 9999 })
 
-  const { mutate: createRole } = useCreateRole()
+  const { mutate: createRole, isLoading: isLoadingCreateRole } = useCreateRole()
```

Comment:
I guess the naming `isCreatingRole` is better than `isLoadingCreateRole` 😬 Same for the other occurrences.

Reply:
Good point ;) I changed it. You can have a second look :) @qhantom